docs/introduction/friendly-url-breadcrumb-navigation.md

```diff
@@ -34,3 +34,18 @@ The rest of the work is done automatically and URLs provided by these providers
 
 *Note: You can use [`UrlListType` in your forms](https://github.com/shopsys/shopsys/blob/master/docs/introduction/using-form-types.md#urllisttype) to edit friendly URLs of existing entities.*
 *If you're interested, you can take a look at the processing of `Article` entity (see `ArticleFacade`, `ArticleData::$urls`, `ArticleDataFactory` and `ArticleFormType`), which allows for this functionality.*
+
+# Breadcrumb navigation
+
+All frontend routes include breadcrumb navigation in the top of the page to ease the navigation for your customers across your e-commerce site.
+When adding a new page on the frontend, you need to implement new `BreadcrumbGenerator` for the new routes to tell the application how the navigation should be displayed.
```

Comment:
I would add a note that this article covers only breadcrumb navigation on front-end and that administration breadcrumb navigation is generated from the [Administration menu](https://github.com/shopsys/shopsys/blob/master/docs/administration/administration-menu.md).

I would also consider renaming the article to *Front-end Breadcrumb Navigation*.

Reply:
Done